### ISSUE TITLE
[Change] Hardcoded loottables for mobs and blocks

### ISSUE THREAD

**Issue by lxly9** [quiqueck/BCLib#58]
### Description

Mods such as EMI loot cannot display any information about drops and conditions for blocks and entities due to them being hardcoded. Tested with BetterX mods.

**Comment by fzzyhmstrs** [comment]
Seems to affect any block-adding mod that uses BCLib, like Eden Ring. All of the block and mob loot tables are entirely incompatible with anything that wants to view or modify them (because there are no loot tables, they are hardcoded). 

Current positively known list of affected mods:
* BetterEnd
* BetterNether
* Eden Ring

EmiLoot is one example, LootJs another one for modpack creators that want to modify block/mob drops.

**Comment by quiqueck** [comment]
Yes, most of our blocks use hard-coded loot-tables. This will eventually change in the 1.19.3 releases.
